widgets/nuxeo-date-picker.html

```diff
@@ -175,24 +187,26 @@
       },
 
       _pickerValueChanged: function() {
-        if (this._pickerValue === undefined) {
-          return;
-        }
-        if (this._wasOpened && this._pickerValue && moment(this._pickerValue).isValid()) {
-          this.set('value', moment(this._pickerValue).toJSON());
-          this._wasOpened = false;
+        if (this._pickerValue !== undefined && !this._preventPickerUpdate) {
+          var date = moment(this._pickerValue);
+          if (this._wasOpened && this._pickerValue && date.isValid()) {
```

Comment:
you already validate if `this._pickerValue` exists in the first `if` so you shouldn't need to check here